### ISSUE TITLE
Investigate SQLite initialization failure without .Net 3.5 installed

### ISSUE THREAD

**Issue by BergWerkGIS** [mapbox/mapbox-unity-sdk#509]
* Unity version: all current 2017.x
* Mapbox SDK version: 1.3
* The platform you're building to: most likely affects just Windows
* A description of what you're trying to do:
Trying to run SDK without .Net 3.5 installed results in:
  > Failed to load 'Assets/Mapbox/Core/Plugins/sqlite/x64/sqlite3.dll' with error 'A dynamic link library (DLL) initialization routine failed.
* Previous reports
  * #361 
  * #496
* Steps to recreate the bug if appropriate:
  * setup a vanilla system with just Unity and no .Net 3.5 Framework

**Comment by JorgenEngenNapstad** [comment]
Hi. Any progress with this issue? It would be good not having to force/instruct our end-users to install .Net 3.5?

**Comment by Naphier** [comment]
Yeah, this is a very bad sticking point for many of our military customers. They are not allowed to install certain things on their PCs. It's been a long time and complete silence on this.

**Comment by eabase** [comment]
BUMP!

Having to rely on a **16 year old** .NET Framework package should be considered a security issue in itself.  If, you're not able to remove this old bloat, from main components, I suggest you fork the project into something like LTS for (defense?) industry and standard. 

Unless there is a better workaround than `.NET 3.5`? 
It seem to rely on an old version of `sqlite3.dll` which could also be/become a security issue.

SQLite3 is very well maintained, so if this is the only dependency, there must be a better solution to integrate this. This is suggested in #1834
